src/utils.js

```diff
@@ -309,7 +310,8 @@ async function printFilteredActionLogs (runtime, logger, limit, filterActions =
   for (let i = (activations.length - 1); i >= 0; i--) {
     const activation = activations[i]
     lastActivationTime = activation.start
-    if (lastActivationTime > startTime) {
+    const shouldPrintLogs = cleanLogs ? startTime && startTime < lastActivationTime : startTime < lastActivationTime
```

Comment:
That was quick @florind12 . While this works just fine, could you also explore achieving this by setting the startTime ? I think there is a `since` parameter for activations.list() . It looks like a slightly cleaner way in case we don't bump into issues with timezones.

Reply:
@Himavanth are the updates good to go?